packages/@okta/vuepress-site/docs/reference/hook-best-practices/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hook best practices
+excerpt: >-
+  Understand the best practices at Okta to implement and secure an Okta Event Hook or Inline Hook.
+---
+
+# Hook best practices
```

Comment:
Should "Hook" be pluralized throughout? It seems odd to refer to it as a 'hook' when pretty much everywhere else we refer to it in the plural form.

Reply:
Yes - I believe Hooks should be plural. In our product docs and within the feature itself on the Admin console, it's always used in the plural sense when referring to the general feature unless it's being using to refer to a single hook, ie, "Add an Inline Hook"

Reply:
I did consciously change this, I think, to adhere to another style I saw. But no problem to go back to the plural form. Will also update file name.